docs/detections/alert-suppression.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[alert-suppression]]
+== Suppress detection alerts
+
+.Requirements and notices
+[sidebar]
+--
+Alert suppression requires a https://www.elastic.co/pricing[Platinum or higher subscription].
+
+preview::[]
+--
+
+Alert suppression allows you to reduce the number of repeated or duplicate detection alerts created by a custom query type detection rule each time the rule runs. 
+
+Normally, when a rule matches multiple source events, multiple alerts are created, one for each event. With alert suppression configured, matching events are grouped by a specified field, and only one alert will be created for each group of events per rule execution. You can also specify multiple fields to group events by unique combinations of values.
+
+Suppressed alerts are indicated in the Alerts table and the alert details flyout, and you can view the original events for suppressed alerts by investigating the alert in Timeline.
+
+NOTE: Alert suppression is not available for Elastic prebuilt rules. If you want to suppress alerts for a prebuilt rule, you can duplicate it, then configure alert suppression on the duplicated rule.
+
+=== Configure alert suppression
+
+You can configure alert suppression when you create or edit a <<create-custom-rule,custom query rule>>. When configuring the rule type (the *Define rule* step for a new rule, the *Definition* tab for an existing rule), enter one or more field names in *Suppress Alerts By*. Configure other rule settings, then save and enable the rule.
+
+TIP: Use the *Rule preview* before saving the rule to visualize how alert suppression will affect the alerts created, based on historical data.
+
+=== Confirm suppressed alerts
+
+The {security-app} has several indicators of whether a detection alert was created using alert suppression, and how many duplicate alerts were suppressed.
```

Comment:
```suggestion
The {security-app} has several indicators of whether a detection alert was created despite alert suppression, and how many duplicate alerts were suppressed.
```
Seems odd to say an alert was created using alert suppression, since suppression reduces the number of alerts. I think it gets the point across though, and I'm not sure "despite" is more clear, so feel free to leave it as is.